src/api/mock_cabinet.js

```diff
@@ -0,0 +1,36 @@
+// Cabinet.vue
+// import { getCabinet } from './api'
+// const cabinet = await getCabinet('perikatan nasional')
+
+//Not needing to re-call the API, we will find politicians based on the following methods
+//Array.prototype.find()
+//Array.prototype.find(x => x.id === id)
+
+const mock_cabinet = {
+    ministries: [
+      {
+        id: 'sadfsadfawefdsfasw4erfsdsdf',
+        name: 'Top Leadership',
+        primary_politician_id: '-L6lZpZcWuJ8siJ3B-yt', // Mahathir
+        primary_title: 'Prime Minister',
+        secondary_politician_ids: [
+          '-LAEFHbjgD01SgG-wwqW',  // Chong Chien Jeng
+          '-L6lYpUWjUHYBOb_fkCB'  // Lim Guan Eng
+        ],
+        secondary_title: 'Senior Minister'
+      },
+      {
+        id: 'dsffsdfsdfsdfsdfsddsfsdfsdfsdf',
+        name: 'Ministry of Education',
+        primary_politician_id: '7TcN9D2JhoYUH1bfS8er',  // Muhyiddin Yassin
+        primary_title: 'Minister of Education',
+        secondary_politician_ids: [
+          '-LAEDr85AyrHvQOBeLJu',  // Abang Jo
+          '31ddGKthhfXp8onAIeQ2'  // Ahmad Faizal Azumu
+        ],
+        secondary_title: 'Deputy Minister'
+      }
+  ]
+}
+
+export { mock_cabinet };
```

Comment:
export default mock_cabinet

Reply:
// e.g. of named export
<script>
export { mock_cabinet } from './mock_cabinet'
</script>

// e.g. of default export
<script>
export mock_cabinet from './mock_cabinet'
</script>

default export is more appropriate